indexer/services/comlink/src/controllers/api/v4/transfers-controller.ts

```diff
@@ -165,6 +179,7 @@ class TransfersController extends Controller {
             ? createdBeforeOrAtHeight.toString()
             : undefined,
           createdBeforeOrAt,
+          page,
         },
         [QueryableField.LIMIT],
         {
```

Comment:
Can you change the ordering to be by eventId desc here? To make transfer ordering deterministic if they occurred at the same height.

Reply:
Done https://github.com/dydxprotocol/v4-chain/pull/1420/commits/2fd8d09f2b96b3cb0e71f79b02c10d7b8bba992e#diff-02ea2aa83d63cb59bf5894f9412f98d5c06e986cb5827f1a757a421173972474R192